openlibrary/templates/type/edition/view.html

```diff
@@ -150,7 +150,10 @@
       </div>
 
       <div class="work-line">
-        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title</a>
+        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title
+          $if work.subtitle:
+            $:(':') $work.subtitle
```

Comment:
To avoid the extra space:
```suggestion
        $_("An edition of") <a href="$work.key?edition=$(ocaid)">$work_title$cond(work.subtitle, ': %s' % work.subtitle)
```